src/Symfony/Component/Routing/Matcher/RedirectableUrlMatcher.php

```diff
@@ -32,9 +32,9 @@ public function match($pathinfo)
             }
 
             try {
-                parent::match($pathinfo.'/');
+                $parameters = parent::match($pathinfo.'/');
 
-                return $this->redirect($pathinfo.'/', null);
+                return array_replace($parameters, $this->redirect($pathinfo.'/', isset($parameters['_route']) ? $parameters['_route'] : null));
```

Comment:
The PR doesn't need to be merged: a local composer repository is made on travis to simulate the merge before it really happen. See .github/build-packages.php